packages/forklift-console-plugin/src/modules/Plans/views/details/tabs/Resources/OpenshiftPlanResources.tsx

```diff
@@ -52,22 +68,38 @@ export const OpenshiftPlanResources: React.FC<{ planInventory: OpenshiftVM[] }>
             <Td width={10}>
               <strong>{t('Virtual machines:')}</strong>
             </Td>
-            <Td width={10}>{planInventory?.length}</Td>
-            <Td width={10}>{planInventoryRunning?.length}</Td>
+            <Td width={10}>{toDecAlignedFormat(planInventory?.length)}</Td>
+            <Td width={10}>{toDecAlignedFormat(planInventoryRunning?.length)}</Td>
           </Tr>
           <Tr>
             <Th width={10}>
-              <strong>{t('Total CPU count:')}</strong>
+              <strong>{t('Total CPU count (Cores):')}</strong>
             </Th>
-            <Td width={10}>{missingCPUInfo ? '-' : `${totalResources.cpuCount} Cores`}</Td>
-            <Td width={10}>{missingCPUInfo ? '-' : `${totalResourcesRunning.cpuCount} Cores`}</Td>
+            <Td width={10}>
+              {missingCPUInfo
+                ? toDecAlignedFormat('-')
+                : toDecAlignedFormat(totalResources.cpuCount)}
+            </Td>
+            <Td width={10}>
+              {missingCPUInfo
+                ? toDecAlignedFormat('-')
+                : toDecAlignedFormat(totalResourcesRunning.cpuCount)}
+            </Td>
           </Tr>
           <Tr>
             <Th width={10}>
-              <strong>{t('Total memory:')}</strong>
+              <strong>{t('Total memory (MB):')}</strong>
```

Comment:
my concern here is that we may want to show GB and TB, for examle the total migration migrate 124 GiB and the running vms use 124 MiB, can you think of a fix that will allow different units ?